app/server.js

```diff
@@ -36,8 +36,10 @@ app.post("/signup", (req, res) => {
                 })
                 .catch((error) => {
                     // insert failed
-                    console.log(error);
-                    res.status(500).send();
+                    if (error.detail === `Key (username)=(${username}) already exists.`) {
+                        return res.status(401).send("Username is already taken.");
```

Comment:
Depending on the error from postgres, we will let the user know they can't sign up with that username